[2024-05-07 16:38:35,285] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: update_visualization.run_spark manual__2024-05-07T16:38:28.927867+00:00 [queued]>
[2024-05-07 16:38:35,301] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: update_visualization.run_spark manual__2024-05-07T16:38:28.927867+00:00 [queued]>
[2024-05-07 16:38:35,301] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-05-07 16:38:35,301] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-05-07 16:38:35,301] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-05-07 16:38:35,322] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): run_spark> on 2024-05-07 16:38:28.927867+00:00
[2024-05-07 16:38:35,335] {standard_task_runner.py:52} INFO - Started process 4938 to run task
[2024-05-07 16:38:35,347] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'update_visualization', 'run_spark', 'manual__2024-05-07T16:38:28.927867+00:00', '--job-id', '174', '--raw', '--subdir', 'DAGS_FOLDER/UpdateVisualization.py', '--cfg-path', '/tmp/tmpvczzyrbo', '--error-file', '/tmp/tmpvbb71g_l']
[2024-05-07 16:38:35,350] {standard_task_runner.py:80} INFO - Job 174: Subtask run_spark
[2024-05-07 16:38:35,466] {task_command.py:369} INFO - Running <TaskInstance: update_visualization.run_spark manual__2024-05-07T16:38:28.927867+00:00 [running]> on host b29e3e3252f3
[2024-05-07 16:38:35,593] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=update_visualization
AIRFLOW_CTX_TASK_ID=run_spark
AIRFLOW_CTX_EXECUTION_DATE=2024-05-07T16:38:28.927867+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-07T16:38:28.927867+00:00
[2024-05-07 16:38:35,595] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-05-07 16:38:35,596] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python /opt/***/dags/spark.py']
[2024-05-07 16:38:35,620] {subprocess.py:85} INFO - Output:
[2024-05-07 16:38:35,929] {subprocess.py:92} INFO - Traceback (most recent call last):
[2024-05-07 16:38:35,933] {subprocess.py:92} INFO -   File "/opt/***/dags/spark.py", line 1, in <module>
[2024-05-07 16:38:35,933] {subprocess.py:92} INFO -     import findspark
[2024-05-07 16:38:35,934] {subprocess.py:92} INFO - ModuleNotFoundError: No module named 'findspark'
[2024-05-07 16:38:35,937] {subprocess.py:96} INFO - Command exited with return code 1
[2024-05-07 16:38:36,032] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 195, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-05-07 16:38:36,050] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=update_visualization, task_id=run_spark, execution_date=20240507T163828, start_date=20240507T163835, end_date=20240507T163836
[2024-05-07 16:38:36,124] {standard_task_runner.py:97} ERROR - Failed to execute job 174 for task run_spark (Bash command failed. The command returned a non-zero exit code 1.; 4938)
[2024-05-07 16:38:36,174] {local_task_job.py:156} INFO - Task exited with return code 1
[2024-05-07 16:38:36,382] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
